genesis/example/doc.go

```diff
@@ -37,5 +37,25 @@ Entities:
 		factory := NewMemberFactory(memberDomain)
 		m, error := factory.Create(parent)
 
+////
+
+    WalletDomain - domain that allows to add new wallets to system
+
+	Usage:
+
+		factory := NewWalletDomainFactory( factoryParent )
+		wDomain, err := factory.Create( domainParent )
+
+		// since Create inject composite to member proxy
+		err := wDomain.CreateWallet( memberProxy )
+
+	Wallet - smart contract that represent wallet
+
+	Usage:
+		factory := NewWalletFactory(walletDomain)
+		w, err := factory.Create(parent)
```

Comment:
We didn't add any examples of working with wallet as with composite. Should be provide it?

Reply:
add example for getting user's balance